pulp_smash/tests/ostree/api_v2/test_copy.py

```diff
@@ -0,0 +1,79 @@
+# coding=utf-8
+"""Test copy of specific content from/to OSTree repositories."""
+import unittest
+from urllib.parse import urljoin
+
+from pulp_smash import api, config, utils
+from pulp_smash.constants import OSTREE_BRANCH, OSTREE_FEED, REPOSITORY_PATH,\
+    OSTREE_FEED_SMALL
+from pulp_smash.tests.ostree.utils import gen_distributor, gen_repo
+from pulp_smash.tests.ostree.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+
+
+class CopyTestCase(unittest.TestCase):
+    """Test copy of specific content from/to OSTree repositories."""
+
+    def test_all(self):
+        """Test copy of specific content from/to OSTree repositories.
+
+        Verify that:
+
+        * When doing a copy from one OSTree repo to another, just certain units
+        in the repo get copied, based on what filters were supplied.
+        """
+        repos = []  # list to store data related to created repos
+        cfg = config.get_config()
+        client = api.Client(cfg, api.json_handler)
+
+        # Create the 1st repository with at least 2 content units
+        body = gen_repo()
+        body['importer_config']['feed'] = OSTREE_FEED_SMALL
+        body['importer_config']['branches'] = ['rawhide', 'stable']
+        body['distributors'].append(gen_distributor())
+        repo = client.post(REPOSITORY_PATH, body)
+        repos.append(repo)
+        self.addCleanup(client.delete, repo['_href'])
+
+        # Sync the 1st repository
+        utils.sync_repo(cfg, repo)
+        first_unit_first_repo = utils.search_units(
+            cfg, repos[0], {})[0]['metadata']['_id']
+        second_unit_first_repo = utils.search_units(
+            cfg, repos[0], {})[1]['metadata']['_id']
+
+        # Create the 2nd repository
+        body = gen_repo()
+        body['importer_config']['feed'] = OSTREE_FEED
+        body['importer_config']['branches'] = [OSTREE_BRANCH]
```

Comment:
Maybe there should be an `OSTREE_BRANCHES` constant instead of an `OSTREE_BRANCH` constant?